src/main/resources/templates/browse.html

```diff
@@ -77,17 +77,15 @@ <h1>Filter</h1>
 <!--Will populate with movies-->
     <div class="container text-center mt-5">
         <div class="row">
-            <div class="column">
-                <h3>Movies</h3>
-                <ul class="list">
-                    <!--Could make the listed titles link to a separate page that can suggest movies based on the selected title?-->
-                    <li class="list-item" th:each="movie : ${movies}">
-                        <h5 th:text="${movie.title}">Title</h5>
-                        <p th:text="${movie.description}">Description</p>
-                        <p>Rating: <span th:text="${movie.rating}"></span></p>
-                        <p>Release Date: <span th:text="${movie.releaseYear}"></span></p>
-                    </li>
-                </ul>
+            <div class="col-md-4 mb-4" th:each="movie: ${movies}">
+                <div class="card" style="width: 18rem;">
+                <div class="card-body">
+                    <h5 class="card-title" th:text="${movie.title}">Movie Title</h5>
+                    <p class="card-text" th:text="${movie.description}">Description</p>
+                    <p>Rating:<span th:text="${movie.rating}"></span></p>
+                    <p>Release Year:<span th:text="${movie.releaseYear}"></span></p>
+                </div>
+                </div>
             </div>
```

Comment:
I have made some UI enhancement to display the list of movies which lacked a
proper structure. I used bootstrap to display each movie in a card to make it look
structured and organized.